addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js

```diff
@@ -111,7 +111,14 @@ FormRenderer.include({
      * @private
      */
     async _mountChatterContainerComponent() {
-        await this._chatterContainerComponent.mount(this._chatterContainerTarget);
+        try {
+            await this._chatterContainerComponent.mount(this._chatterContainerTarget);
+        } catch (error) {
+            if (error.message === "Mounting operation cancelled") {
+                // do nothing
+            }
+            throw error;
+        }
```

Comment:
```suggestion
        try {
            await this._chatterContainerComponent.mount(this._chatterContainerTarget);
        } catch (error) {
            if (error.message !== "Mounting operation cancelled") {
                throw error;
            }
        }
```
Isn't that what we want?
The catch seems useless if we always throw again.